src/net/atlas/db.rs

```diff
@@ -231,30 +257,76 @@ impl AtlasDB {
         Ok(res)
     }
 
-    pub fn insert_new_attachment(&mut self, attachment: &Attachment) -> Result<(), db_error> {
+    pub fn insert_uninstanciated_attachment(
```

Comment:
"uninstantiated"

Reply:
Addressed in https://github.com/blockstack/stacks-blockchain/pull/2204/commits/fe89bce9fa000b3aad271fa6ae5404145cafa1f7